arcoirsl_webstore/database/migrations/2023_01_30_163105_create_products_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
```

Comment:
using ""Column Modifiers"

https://laravel.com/docs/9.x/migrations#column-modifiers

Make the table look safer, avoid **null** situations,

some example:

`
->default($value)  -- Specify a "default" value for the column.
`
`
->nullable($value = true) -- Allow NULL values to be inserted into the column.
`
`
->unsigned() -- Set INTEGER columns as UNSIGNED (MySQL).
`

and some index types
https://laravel.com/docs/9.x/migrations#available-index-types

`
$table->primary('id'); Adds a primary key.
`
`
$table->unique('email'); Adds a unique index.
`